op-service/flags/flags.go

```diff
@@ -55,10 +55,11 @@ func CLIRollupConfigFlag(envPrefix string) cli.Flag {
 // This checks flags that are exclusive & required. Specifically for each
 // set of flags, exactly one flag must be set.
 var requiredXorFlags = [][]string{
-	{
-		RollupConfigFlagName,
-		NetworkFlagName,
-	},
+	// TODO(https://github.com/ethereum-optimism/client-pod/issues/391): Re-enable this check at a later point
```

Comment:
I think the todo checker requires
```suggestion
	// TODO(client-pod#391): Re-enable this check at a later point
```